mne/source_estimate.py

```diff
@@ -33,41 +33,42 @@
 
 def _read_stc(filename):
     """Aux Function."""
-    fid = open(filename, 'rb')
+    with open(filename, 'rb') as fid:
+        buf = fid.read()
 
     stc = dict()
-
-    fid.seek(0, 2)  # go to end of file
-    file_length = fid.tell()
```

Comment:
yes, it was reading in the entire file. It probably depends on which filesystem you use. I'm using a mounted NFS share.